EmployeesSalaries/EmployeesSalaries/Models/SalaryCalculaters/DevSalary.cs

```diff
@@ -0,0 +1,11 @@
+﻿using EmployeesSalaries.Models.Interfaces;
+
+namespace EmployeesSalaries.Models
+{
+    public class DevSalary : IEmployeeSalaryCalculater
+    {
+        public double BasicSalary { get; set; } = 800;
```

Comment:
setting the salary in the class to 800 does not make sense. you should add all the data in different static class to include all you're system data. consider it as database.
also, the salary should be decimal not double.